sirepo/job_driver/docker.py

```diff
@@ -124,10 +124,6 @@ async def kill(self):
             await self._cmd(
                 ("stop", "--time={}".format(job_driver.KILL_TIMEOUT_SECS), self._cname),
             )
-        except sirepo.util.ASYNC_CANCELED_ERROR:
```

Comment:
This is an implicit coupling on the meaning of ASYNC_CANCELED_ERROR. concurrent.futures.CancelledError is a subclass of Exception so this changes the semantics. Is concurrent.futures.CancelledError not called in 3.10? If it is, then we need to handle that case specially to re-raise like it was. 

I think ASYNC_CANCELED_ERROR belongs in const, but it could be that it should be removed and carefully coded in all cases.

Reply:
2f7887c01